tikv-configuration-file.md

```diff
@@ -146,24 +146,24 @@ Configuration items related to storage thread pool.
 ### `use-unified-pool`
 
 + Determines whether to use the unified thread pool (configured in [`readpool.unified`](#readpoolunified)) for storage requests. If the value of this parameter is `false`, a separate thread pool is used, which is configured through the rest parameters in this section (`readpool.storage`).
-+ Default value: `true`
++ Default value: If this section (`readpool.storage`) has no other configurations, the default value is `true`. Otherwise, to upgrade compatibility, the default value is `false`. Change the configuration in [`readpool.unified`](#readpoolunified) as needed before enabling this option.
```

Comment:
```suggestion
+ Default value: If this section (`readpool.storage`) has no other configurations, the default value is `true`. Otherwise, for the forward compatibility, the default value is `false`. Change the configuration in [`readpool.unified`](#readpoolunified) as needed before enabling this option.
```